projects/igniteui-angular/src/lib/grids/row.directive.ts

```diff
@@ -367,7 +367,7 @@ export class IgxRowDirective<T extends IgxGridBaseDirective & GridType> implemen
      */
     public update(value: any) {
         const crudService = this.crudService;
-        if (crudService.inEditMode && crudService.cell.id.rowID === this.rowID) {
+        if (crudService.cellInEditMode && crudService.cell.id.rowID === this.rowID) {
```

Comment:
Unrelated to the change, but if there's an active cell being edited the update will discard, but if it's just the row (not the most common scenario, I know) it stays in edit?